content/en/post/2024-11-trumanitarian-ep-92/index.md

```diff
@@ -0,0 +1,69 @@
+---
+#widget: pages
+#headless: true
+title: "Neil Smyth on Trumanitarian: The Alchemist"
+date: "2024-11-12"
+lastmod: "2024-11-12"
+math: false
+diagram: false
+summary: "Our CEO, Neil Smyth, discusses the societal impact of profit-driven digital platforms and Alkemio's mission to offer an alternative on the Trumanitarian podcast."
+authors:
+  - rosso-francesco
+  - smyth-neil
+tags:
+  - digital platforms
+  - collective action
+  - social impact
+  - open source
+  - steward ownership
+#categories:
+  #- Platform
+
+featured: false
+draft: false
+image:
+  placement: 1
+  preview_only: true
+
+---
+![](./header.png)
+
+“Imagine building a house, but you don’t own the pipes, nor the foundations, and others can decide who communicates with who.”
+
+This is how our CEO, Neil Smyth, explains what is happening in society due to the pervasiveness of profit-maximizing digital platforms.
+
+Neil was recently invited to discuss this issue and Alkemio’s mission to offer an alternative on Episode 92 of the Trumanitarian podcast, hosted by Lars Peter Nissen. In this blog post, we highlight some of the key themes.
+
+### The Problem with the Digital Status Quo
+
+Neil addresses a critical issue in today’s digital landscape: profit-driven platforms control the very infrastructure of our online spaces. With the analogy of building a house but not owning the foundations, Neil illustrates that this isn't a hypothetical future; it’s our present reality. This phenomenon, often referred to as "techno-feudalism," is already here.
+
+### Introducing Alkemio: Collective Action Reimagined
+
+Alkemio was designed to enable collective action, providing safe digital spaces for collaborative work, backed by an organization dedicated to societal benefit.
+
+A significant milestone for Alkemio was its adoption of Steward Ownership. This commitment assures that Alkemio’s purpose remains central: the venture cannot be sold, and profit serves purpose. This diverges starkly from the prevailing model of digital platforms, where shareholder profits often take precedence over societal interests.
+
+Additionally, Alkemio’s Open Source commitment guarantees a transparent exit option for users. While Alkemio hopes to sustain its vision through a commercially hosted service, the open-source promise means users can leave if necessary.
```

Comment:
On the second aspect of open source (in addition to transparency), what are your thoughts on framing it not as users leaving, however, rather as complete freedom and ability for users to run their instance? The difference in this argument is that open source is not so much a freedom to exit, but a freedom to build and maintain.

Reply:
Agreed, much more positive framing